pkg/go/transformer/dsltojson.go

```diff
@@ -372,7 +372,17 @@ func (c *OpenFgaDslErrorListener) SyntaxError(recognizer antlr.Recognizer, offen
 func ParseDSL(data string) (*OpenFgaDslListener, *OpenFgaDslErrorListener) {
 	cleanedLines := []string{}
 	for _, line := range strings.Split(data, "\n") {
-		cleanedLines = append(cleanedLines, strings.TrimRight(line, " "))
+		cleanedLine := ""
```

Comment:
Is this something that could feasibly be moved to the lexer/parser?

Reply:
Ideally, yeah. But we've had a hard time with that b/c `user#follower` is valid syntax that should be parsed.

Reply:
Ideally, yeah. But we've had a hard time with that b/c `user#follower` is valid syntax that should be parsed.

Reply:
Thanks for the context!